logics/impls/controller/mod.rs

```diff
@@ -1214,7 +1402,9 @@ impl<T: Storage<Data>> Internal for T {
         caller_pool: Option<(AccountId, PoolAttributes)>,
     ) -> Result<(U256, U256)> {
         // For each asset the account is in
-        let account_assets = self._account_assets(account, token_modify);
```

Comment:
I would like to see a comment in the code comments for future maintenance developers as to why _account_assets is not used in NOTE:.
I understand that you are avoiding the Pool call on the _account_assets side and doing a similar check subsequent to this function.